src/lib.rs

```diff
@@ -2,6 +2,66 @@
 // See the COPYRIGHT file at the top-level directory of this distribution.
 // Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
 
+//! # Cairo bindings
+//!
+//!   This library contains safe Rust bindings for [Cairo](https://www.cairographics.org/).
+//!   It is a part of [Gtk-rs](http://gtk-rs.org/).
+//!
+//! ## Crate features
+//!
+//! ### Default-on features
+//!
+//! * **use_glib** -
+//!   Use with [glib](https://gtk-rs.org/docs/glib/)
+//!
+//! ### Fileformat features
+//!
+//! * **png** -
+//!   Reading and writing PNG images
```

Comment:
Why putting it on a newline?

Reply:
No particular reason, I can remove them as well.

Reply:
Yes please!

Reply:
What I meant was to write it this: `* **png** - Reading and writing PNG images`

Reply:
Done!
It looks indeed better